composer.json

```diff
@@ -7,7 +7,7 @@
         "sonata",
         "classification"
     ],
-    "homepage": "https://sonata-project.org/bundles/classification-media",
+    "homepage": "https://github.com/sonata-project/SonataClassificationMediaBundle",
```

Comment:
We should remove documentation from README.md or add it to docs.sonata-project.org in Abandoned/Deprecated section.

Also look at (it is docs for EasyExtendsBundle instead):
https://docs.sonata-project.org/projects/SonataClassificationMediaBundle/en/0.x/

Reply:
Done